pkg/config/raw_meta_build.go

```diff
@@ -1,9 +1,10 @@
 package config
 
 type rawMetaBuild struct {
-	Platform []string `yaml:"platform,omitempty"`
-	Staged   bool     `yaml:"staged,omitempty"`
-	rawMeta  *rawMeta
+	Platform     []string      `yaml:"platform,omitempty"`
+	Staged       bool          `yaml:"staged,omitempty"`
+	RawImageSpec *rawImageSpec `yaml:"imageSpec,omitempty"`
```

Comment:
I don’t see any real use cases for supporting all directives at the global level. Only author, resetHistory, and config.labels/removeLabels seem reasonable.